apps/remix-ide/src/app/files/fileProvider.js

```diff
@@ -63,11 +63,11 @@ class FileProvider {
     })
   }
 
-  exists (path, cb) {
+  async exists (path) {
```

Comment:
why do you need to add `async` if the content of the function is not using `await` ?

Reply:
Adding async returns a promise. I added it because fileManager expects a promise just like how remixdProvider returns a promise.